SPECS/sm.spec

```diff
@@ -35,8 +35,8 @@ Obsoletes: sm-additional-drivers
 
 # XCP-ng patches
```

Comment:
The comment above must be removed. It would also be good to document how you created the v2.30.3-xcp-ng tag and the 2.30.3-8.2 branch. This way, next time we must do it, we'll remember the process.